src/tokens/evo-dark.css

```diff
@@ -41,7 +41,9 @@
         --color-state-inverse-hover: #f7f7f7;
         --color-state-inverse-active: #f8f8f8;
         --color-state-accent-hover: #6ba2ff;
+        --color-state-accent-button-hover: #6ba2ff;
```

Comment:
It's named `--color-state-accent-button-hover' here but '--color-state-button-accent-hover' in light.css